scripts/lib/CIME/SystemTests/ncr.py

```diff
@@ -1,111 +1,61 @@
-        restore("env_build.NCR1.xml", newname="env_build.xml")
-        shutil.copy("{}/{}.exe.NCR1".format(exeroot, cime_model),
-                    "{}/{}.exe".format(exeroot, cime_model))
-
-
-        stop_n      = self._case.get_value("STOP_N")
-        stop_option = self._case.get_value("STOP_OPTION")
-
-        self._case.set_value("HIST_N", stop_n)
-        self._case.set_value("HIST_OPTION", stop_option)
-        self._case.set_value("CONTINUE_RUN", False)
-        self._case.set_value("REST_OPTION", "none")
-        self._case.flush()
-
-        #======================================================================
-        # do an initial run test with NINST 1
-        #======================================================================
-        logger.info("default: doing a {} {} with NINST1".format(stop_n, stop_option))
-        self.run_indv()
-
-        #======================================================================
-        # do an initial run test with NINST 2
-        # want to run on same pe counts per instance and same cpl pe count
-        #======================================================================
-
-        os.remove("{}/{}.exe".format(exeroot, cime_model))
-        shutil.copy("{}/{}.exe.NCR2".format(exeroot, cime_model),
-                    "{}/{}.exe".format(exeroot, cime_model))
-        restore("env_build.NCR2.xml", newname="env_build.xml")
-
-        logger.info("default: doing a {} {} with NINST2".format(stop_n, stop_option))
-        self.run_indv(suffix="multiinst")
-
-        # Compare
-        self._component_compare_test("base", "multiinst")
+        SystemTestsCompareTwo.__init__(self, case,
+                                       separate_builds = True,
+                                       run_two_suffix = "singleinst",
+                                       run_one_description = "default build",
+                                       run_two_description = ("half the number of tasks, " +
+                                                              "twice the number of instances"))
+
+    def _case_one_setup(self):
+        # Set the number of instances, the ROOTPEs, and the number of tasks
+        # The second case should have twice the number of instances and half the number of tasks
+        # All tasks should be running concurrently
+        comp_classes = self._case.get_values("COMP_CLASSES")
+        # Is this correct?
+        comp_classes.remove("CPL")
+
+        ntasks_sum = 0
+        # ['ATM','OCN','WAV','GLC','ICE','ROF','LND']
+        for comp in comp_classes:
+            self._case.set_value("NINST_{}".format(comp), str(1))
+            self._case.set_value("ROOTPE_{}".format(comp), ntasks_sum)
+            ntasks = self._case.get_value("NTASKS_{}".format(comp))
+            ntasks_sum += ntasks * 2
+            self._case.set_value("NTASKS_{}".format(comp), ntasks * 2)
+        case_setup(self._case, test_mode = False, reset = True)
```

Comment:
If the two test runs have different pe counts as this one does then the run with the larger value should write the case.test file and the other one should not overwrite it.  That way the batch system will allocate enough resources for the test.